### PR TITLE
Add documentation pages with dummy data

### PR DESCRIPTION
**Pull request by Peartes**
This commit adds the documentation pages with dummy data addressing the issue [here](https://www.notion.so/curvelabs/339a249185e14ddcbd731e5ba07c46c7?v=f7cba822bb9043a5a87ab80b5e2b40ef&p=f8d4d82cd2654325a4e47194b84501bd)

### CODE REVIEW

File: src/documentation/documentation.html [hunk 1]
```diff
@@ -2,10 +2,12 @@
   <div class="page animated-page au-animate documentsContainer">
     <div class="sidebar">
       <template repeat.for="doc of router.navigation">
-        <a href.to-view="doc.href"
-          class="document ${doc.isActive ? 'active' : ''}">
-          ${doc.title}
-        </a>
+        <div>
```
Comment (dkent600) [inline src/documentation/documentation.html:5]: better than adding an unnecessary div is to add `display: block` to the `a` (in the scss)

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/documentation/documentation.scss [hunk 2]
```diff
@@ -1,2 +1,38 @@
+@import "styles/styles.scss";
 .documentsContainer {
```
Comment (dkent600) [inline src/documentation/documentation.scss:2]: The layout of the page does not conform to the doc.  (For example: Is missing the header)

Comment (dkent600) [reply src/documentation/documentation.scss:2]: still don't have it

image

Comment (Peartes) [reply src/documentation/documentation.scss:2]: This is what the branch currently has @dkent600 

screenshot

Comment (Peartes) [reply src/documentation/documentation.scss:2]: This is what the branch currently has 
Screenshot 2021-06-02 at 10 37 07

Comment (Peartes) [reply src/documentation/documentation.scss:2]: The layout now matches. Please re-review

Comment (dkent600) [reply src/documentation/documentation.scss:2]: Doesn't match.  There should not be a different background color and there is too much column whitespace between "Documentation" and "Title 1"

image

Comment (Peartes) [reply src/documentation/documentation.scss:2]: Okay so here is the adjusted documentation page, should match the mocks now.
Screenshot 2021-06-02 at 12 51 06

File: src/documentation/documentation.scss [hunk 3]
```diff
@@ -1,2 +1,43 @@
...
+            text-decoration: none;
+            display: block;
+        }
+
+        .active {
+            color: $Secondary02;
+        }
+    }
+    .docsContainer {
+        background-color: $BG01;
```
Comment (dkent600) [inline src/documentation/documentation.scss:27]: why is this needed?

**Review by dkent600 — APPROVED** [review]
